app/Commands/GenerateCommand.php

```diff
@@ -30,13 +32,138 @@ class GenerateCommand extends Command
      */
     public function handle()
     {
-        $r = file_put_contents('Dockerfile', (string)view('dockerfile', [
+        // determine the default answer based on the options
```

Comment:
Can you give more description to what is being answered in this comment, like:

```suggestion
        // Determine the default answer on whether to overwrite an existing Dockerfile based on the options
```
But, this is very minor, so feel free to skip

Reply:
Also another minor feel-free-to-skip: it would be nice to have the first letter of the start of each sentence in a comment capitalized